tests/transformers/test_transformers_model_export.py

```diff
@@ -1999,18 +1998,18 @@ def test_infer_signature_from_input_example_only(
     assert model.signature.inputs.to_dict() == in_signature
     assert model.signature.outputs.to_dict() == out_signature
 
-    saved_example = _read_example(model, model_path).to_dict(orient="records")
-    if isinstance(example, str):
-        assert next(iter(saved_example[0].values())) == example
-    elif isinstance(example, list):
-        assert list(saved_example[0].values()) == example
-    else:
+    saved_example = _read_example(model, model_path)
+    # saved example is the same as input example if it is list of scalars
+    if isinstance(example, list):
+        assert (saved_example == example).all()
```

Comment:
we can just do a direct equivalency check here, right?

```python
    assert saved_example == example
``` 
?

Reply:
We can't, saved_example is a numpy array